packages/mongodb-rag-ingest/src/sources/mongodb-university/MongoDbUniversityDataSource.ts

```diff
@@ -44,23 +48,6 @@ export interface MakeMongoDbUniversityDataSourceParams {
   metadata?: PageMetadata;
 }
 
-/**
-  Filter function to only include public, published,
-  and non-legacy MongoDB University content.
-
-  > ⚠️ **Important** ⚠️
-  >
-  > You should include *only* this content or a subset of it
-  > in externally facing applications.
- */
-export const filterOnlyPublicActiveTiCatalogItems: MakeMongoDbUniversityDataSourceParams["tiCatalogFilterFunc"] =
```

Comment:
by using the `public` param instead of this filter func, i want to  be sure that we are doing exactly what we want

worth answering these questions:

1. could we include any other microsites besides university? 
2. are we only capturing published sites, not in-development/legacy sites? 
3. what's behavior where there are >1 videos?

Reply:
I updated the comment to be a bit more descriptive, explaining what publicOnly does on the /catalog/ti endpoint side of things. So 1 & 2 are yes, only University microsite, published, not in development, not legacy. 

Im not sure what you mean by #3. Most contentItems do have an `associated_videos` field with many videos listed. We have some Lab Only courses that wouldn't have any video. And higher level courses like Learning Paths and Course don't have videos, but their nested content does. I believe theres a conditional in place in that would handle no videos within the makeCatalogPages function 
```
        if (lesson.videos.length === 0) {
          continue;
        }
```